src/main.c

```diff
@@ -2182,6 +2182,11 @@ static GOptionEntry entries_encryption[] = {
 	{0}
 };
 
+static GOptionEntry entries_mount[] = {
+	{"pre-check", '\0', 0, G_OPTION_ARG_NONE, &pre_check, "check complete bundle before mounting", NULL},
```

Comment:
Instead of having the `pre-check` entry twice (here and in `entries_install`), you can reuse this list for `install_group` by calling `g_option_group_add_entries()`. See how it's done for `bundle_group`.

Reply:
done